sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -290,7 +289,7 @@ def __init__(self, **kwargs):
 
     @classmethod
     def _from_generated(cls, field, read_result):
-        if field is None or isinstance(field, six.string_types):
+        if field is None or all(field_data is None for field_data in [field.page, field.text, field.bounding_box]):
```

Comment:
Just noticed the documentation for this still says 
>    """Represents the text that is part of a form field.

Do we still want to say it represents text?

Reply:
fair point - i'll update this